src/Camera_TEST.cc

```diff
@@ -62,6 +62,18 @@ TEST(DOMCamera, Construction)
   cam.SetFarClip(200.2);
   EXPECT_DOUBLE_EQ(200.2, cam.FarClip());
 
+  cam.SetSegmentationType("semantic");
+  EXPECT_EQ("semantic", cam.SegmentationType());
+  EXPECT_TRUE(cam.HasSegmentationType());
+
+  cam.SetBoundingBoxType("3d");
+  EXPECT_EQ("3d", cam.BoundingBoxType());
+  EXPECT_TRUE(cam.HasBoundingBoxType());
```

Comment:
Can you also add test coverage for the `SetHasSegmentationType` and `SetHasBoundingBoxType` methods?